scripts/docker/test.py

```diff
@@ -117,9 +117,10 @@ def start_network(sdk_dir, bin_dir, network_dir, config, template):
 
 def run_java_tests(JAVA):
     sys.stdout.flush()
-    #subprocess.check_call(['mvn test -Dcucumber.options="--tags \\"not @crosstest\\""'], shell=True, cwd=JAVA['cucumber'])
+    subprocess.check_call(['mvn test -Dcucumber.options="--tags \\"not @crosstest\\""'], shell=True, cwd=JAVA['cucumber'])
     #subprocess.check_call(['mvn test -Dcucumber.options="--tags @template"'], shell=True, cwd=JAVA['cucumber'])
-    subprocess.check_call(['mvn test -Dcucumber.options="/opt/sdk-testing/features/template.feature"'], shell=True, cwd=JAVA['cucumber'])
+
+    #subprocess.check_call(['mvn test -Dcucumber.options="/opt/sdk-testing/features/template.feature"'], shell=True, cwd=JAVA['cucumber'])
```

Comment:
Do we want to leave in this commented-out code? "Yes" is a perfectly fine answer

Reply:
this script is incomplete, I'm not sure if we'll keep it